packages/@aws-cdk/triggers/lib/lambda/index.ts

```diff
@@ -59,9 +59,14 @@ export async function handler(event: AWSLambda.CloudFormationCustomResourceEvent
   const invocationType = event.ResourceProperties.InvocationType;
   const timeout = event.ResourceProperties.Timeout;
 
-  const invokeResponse = await invoke(handlerArn, invocationType, timeout);
+  const parsedTimeout = parseInt(timeout);
+  if (isNaN(parsedTimeout)) {
+    throw new Error(`The "Timeout" property with value ${timeout} is not a parseable to a number`);
```

Comment:
is not ~~a~~ parseable -> is not parseable

Reply:
Woops, will fix in next commit.